addons/crm/tests/test_crm_lead_convert.py

```diff
@@ -241,6 +242,28 @@ def test_lead_convert_corner_cases_matching(self):
         # TDE FIXME: should take into account normalized email version, not encoded one
         # self.assertEqual(convert.partner_id, customer)
 
+    @users('user_sales_manager')
+    def test_lead_convert_corner_cases_archived_lang(self):
+        """ Test Lead._create_customer() """
```

Comment:
would rather go something like `test_lead_convert_no_lang(self)` and then specify what we do in the docstring like `""" Ensure converting a lead with an archived language correctly falls back on the default partner language. """`

This way the test is more 'use case driven' than technically driven, if that makes sense.

Reply:
updated